frontend/src/styles/LoginRegistrationPage.css

```diff
@@ -1,97 +1,246 @@
-div.loginHeader {
-  font-size: 40px;
-  color: #1DE28F;
-}
+@import url('https://fonts.googleapis.com/css?family=Teko');
 
+:root {
+  --BackgroundColor: #202020;
+  --themeColor: #1DE28F;
+  --bodyColor: var(--baseColor);
+  --FontFamily: 'Teko';
+  --FontSize: 14px;
+  --FontWeight: 300;
+  --LineHeight: 1.5;
+}
+body {
+  background-color: var(--BackgroundColor);
+  color: var(--bodyColor);
+  font-family: var(--FontFamily);
+  font-size: var(--FontSize);
+  font-weight: var(--FontWeight);
+  height: 100%;
+  line-height: var(--LineHeight);
+  margin: 0;
+  min-height: 60px;
+}
+form{
+  padding-top: 60px;
+}
 div.login1-button {
```

Comment:
You changed the login1-button but I don't think it is ever used. Verify and remove this entire block of code if it's not being used.